### PR TITLE
Docs update for dts

### PR DESCRIPTION
**Pull request by MarshallOfSound**
/cc @zeke

### CODE REVIEW

File: lib/method.js [hunk 1]
```diff
@@ -16,6 +16,13 @@ module.exports = class Method extends CollectionItem {
       this.parameters = this.getParameters()
       this.returns = this.getReturns()
 
+      const subProp = this.name.split('.')
```
Comment (MarshallOfSound) [inline lib/method.js]: This recognizes patterns such as `app.dock.show()` and splits it into just the `show()` name and declares it has a "superObject" called "dock"

File: lib/api.js [hunk 2]
```diff
@@ -28,8 +28,23 @@ class API {
     }
 
     if (this.type === 'Module') {
+      const subObjects = {}
       this.events = new Collection('Event', this, '#events', 2)
-      this.methods = new Collection('Method', this, '#methods', 2)
+      this.properties = new Collection('Property', this, `#properties`, 2)
+      this.methods = new Collection('Method', this, '#methods', 2).filter(method => {
```
Comment (MarshallOfSound) [inline lib/api.js]: Here we scan through all methods and if any of them have a "superObject" we remove them from the methods list and put them in the properties list under a new "Object" property with the same name as the given "superObject"

**Comment by zeke** [comment]
Hey sorry this escaped my view. Looks like there's a CI error:

> Error: CrashReport does not appear to have a source document

**Comment by MarshallOfSound** [comment]
@zeke Build is now green 👍

**Pull request merged**